docs/docs/getting-started/components.md

```diff
@@ -73,6 +73,39 @@ Airy Core contains the following core components:
 
 ## Installation
 
-For installation purposes Airy Core is packaged in a Helm chart which creates all the necessary Kubernetes resources. However, every component is an independent entity and can be installed or uninstalled separately. Every component is packaged in its own independent Helm chart and belongs to a `repository`. By default, the `airy-core` repository is added with the components that can be found under `infrastructure/helm-chart/charts/components/charts`.
+For installation purposes Airy Core is packaged in a Helm chart which creates all the necessary Kubernetes resources. However, every component is an independent entity and can be installed or uninstalled separately. Every component is packaged in its own independent Helm chart and belongs to a `repository`. By default, the `airy-core` repository is added with the components that can be found under https://helm.airy.co.
 
-Some components are installed by default, while others can be added or removed later, depending on the particular use case. Components can be managed through the `Control center` or through the [components](/api/endpoints/components) and [repositories](/api/endpoints/repositories) API endpoints.
+You can install or uninstall a component through the `components.install` and `components.uninstall` endpoint respectfully. You can see all the available components either from the `Catalog` in the `Control center` or by accessing the `components.list` endpoint.
+
+The following components are part of `Airy Core` and they cannot be uninstalled:
+
+- airy-controller
+- api-admin
+- communication
+- api-websocket
+- frontend-inbox
+- frontend-control-center
+
+Here is a list of the open source components which can be added to `Airy Core`:
+- api-contacts
+- integration-source-api
+- integration-webhook
+- media-resolver
+- sources-chatplugin
+- sources-facebook
+- sources-google
+- sources-twilio
+- sources-viber
+- sources-whatsapp
+
+However, you can also manage the components as helm charts. For example, to install the `sources-chatplugin` component, run this in your workspace:
```

Comment:
Do we not want users to download components via the Control Center? Or is this still under development? Do we prefer that they use the endpoints over the UI?

Reply:
The `Control Center` is fully functional. I just thought of sharing additional info here, so that it is more clear how we install/uninstall them and as a failback if the UI fails.

Reply:
Ah, I now see what you mean. The `Control center` should come as a first suggestion :). I will add this :+1:.